CryptoPkg/Library/BaseCryptLib/Pk/CryptPkcs7VerifyEku.c

```diff
@@ -451,7 +451,7 @@ VerifyEKUsInPkcs7Signature (
   //
   // Create the PKCS7 object.
   //
-  Pkcs7 = d2i_PKCS7(NULL, (CONST CHAR8 **)&Temp, (INT32)SignedDataSize);
+  Pkcs7 = d2i_PKCS7(NULL, (const unsigned char **) &Temp, (INT32)SignedDataSize);
```

Comment:
Yes, it should be (const unsigned char**) or to be even more UEFI like (const UINT8**) or event better we could take the casting out completely because Temp is declared as UINT8*.